app.py

```
@@ -917,11 +1273,162 @@ def _stop_lifx_batch_sender_thread():
+        try:
+            kind = item[0]
+            device = item[1]
+            if kind == 'nl_zones':
+                _kind, device, zone_cmds, _duration_ms = item
+                nanoleaf_client.send_zones(device, zone_cmds)
+            elif kind == 'nl_color':
+                _kind, device, r, g, b, _kelvin, _duration_ms, brightness = item
+                nanoleaf_client.send_color(device, r, g, b, brightness)
+            else:
+                unreachable: str = kind
+                raise ValueError(f'Unhandled Nanoleaf batch kind: {unreachable}')
```

Ruff (0.16.1)

[warning] Abstract `raise` to an inner function

(TRY301)